frontend/packages/kubevirt-plugin/src/constants/vm/network/network-interface-model.ts

```diff
@@ -4,11 +4,7 @@ import { READABLE_VIRTIO } from '../constants';
 
 export class NetworkInterfaceModel extends ObjectEnum<string> {
   static readonly VIRTIO = new NetworkInterfaceModel('virtio');
-  static readonly E1000 = new NetworkInterfaceModel('e1000');
```

Comment:
they are still valid values (you can still have a VM with these models). Can you please just mark them as not supported? Similar to what we do in DiskType enum? And filter them in places where we use them.

Reply:
using the deprecation logic like in disks :-)